card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+{
+  "url": "http://localhost:3000/export.json",
+  "timestamp": "2015-02-25 14:55:17 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[style: glyphicons]]\n[[simple skin]]\n[[simple bootstrap skin]]\n[[Default Layout]]\n[[*all+*script]]\n[[*page header]]\n[[style: bootstrap compatible]]\n[[Old Layout]]",
+    "value": [
```

Comment:
btw, this is the default json format.  if you have a pointer card named "export" and go to "export.json", this is what you get.